roles/testnode/vars/ubuntu_16.yml

```diff
@@ -0,0 +1,16 @@
+---
+packages:
+  - mpich
+  - qemu-system-x86
+  - lttng-tools
+  ###
+  # for setting BIOS settings
+  ###
+  - libcrypto++9v5
+  ###
+  # used by workunits
+  - mpich
```

Comment:
This is version ``3.2`` and version ``2`` is not available. If a workunit needs version 2 only, it will break. I found that src/jobs/meta1 uses `mpiexec` and src/script/runjob.pl back in hammer also does. None of them are referenced by qa workunits. It looks like (from a distance) that we won't break anything. But since I have no clue what these scripts are supposed to do, maybe I'm missing something.